File: DataStructure_Alogorithm/Algorithm/Inflearn_basic_cpp/29_3의개수/main.cpp
#include<stdio.h>
#include<algorithm>
#include<vector>
using namespace std;

int main() {
	int i, n, tmp, digit, cnt; 
	scanf("%d", &n);

	for (i = 1; i <= n; i++) {
		tmp = i;

		while (tmp > 0) {
			digit = tmp % 10;
			if (digit == 3) cnt++;
			tmp = tmp / 10;
		}
	}

	printf("%d", cnt);


	return 0;

}

File: DataStructure_Alogorithm/Algorithm/Inflearn_basic_cpp/29_3의개수/main.go
package main

import "fmt"

func main() {
	var N, tmp, count int

	fmt.Scanf("%d", &N)

	for i := 1; i <= N; i++ {
		tmp = i
		for tmp > 0 {
			if tmp%10 == 3 {
				count++
			}
			tmp = tmp / 10
		}
	}

	fmt.Println(count)
}
